README.md

```diff
@@ -73,6 +73,41 @@ nailed --jenkins
 * create a `cronjob` for automated data collection with `nailed`
 * start the webserver with `nailed --server`
 
+## Running as a Docker container
+
+* Build the image
+
+```
+$ git clone https://github.com/MaximilianMeister/nailed
+$ cd nailed
+$ docker build -t nailed:latest .
+```
+
+* Create a directory to hold the data, and create the config, db and log subdirectories
+
+```
+mkdir -p /mystorage/config
+mkdir -p /mystorage/log
+mkdir -p /mystorage/db
+``
+
+Add .oscrc and netrc into /mystorage/config.
```

Comment:
`.` missing at netrc
I guess the `config.yml` should also go there? maybe worth mentioning here